README.md

```diff
@@ -89,9 +89,37 @@ deepstate-angr --num_workers 4 --output_test_dir out $DEEPSTATE/build/examples/I
 
 DeepState consists of a static library, used to write test harnesses, and command-line _executors_ written in Python. At this time, the best documentation is in the [examples](/examples) and in our [paper](https://agroce.github.io/bar18.pdf).
 
-## Fuzzing
+## Fuzzing with libFuzzer
+
+If you install a recent-enough clang, and run `cmake` when you install
+with `BUILD_LIBFUZZER` defined, you can generate tests using LlibFuzzer.
```

Comment:
It wasn't clear for me that `BUILD_LIBFUZZER` is a environment variable, not a cmake variable (defined with `-DBUILD_LIBFUZZER=1`).